utils/sb2

```diff
@@ -655,12 +654,12 @@ function write_ld_library_path_replacement_to_exec_config()
 
 	# Include directories listed in ld.so.conf
 	if [ -f $rootdir/etc/ld.so.conf ]; then
-		lloc2=`egrep '^/' $rootdir/etc/ld.so.conf`
+		lloc2=$(grep -E '^/' $rootdir/etc/ld.so.conf)
 		liblocations="$liblocations $lloc2"
 	fi
 	# Include directories listed in ld.so.conf.d/*
 	if [ -d $rootdir/etc/ld.so.conf.d ]; then
-		lloc2=`cat $rootdir/etc/ld.so.conf.d/* 2>/dev/null | egrep '^/'`
+		lloc2=$(grep --extended-regexp --only-matching '^/' $rootdir/etc/ld.so.conf.d/* )
```

Comment:
Are we sure this path always exists? The previous code didn't make that assumption.